src/julia-syntax.scm

```diff
@@ -362,7 +362,8 @@
                                                            'nothing
                                                            (cons 'list (map car sparams)))
                                                       ,(if (null? loc) 0 (cadr loc))
-                                                      (inert ,(if (null? loc) 'none (caddr loc))))))))
+                                                      (inert ,(if (null? loc) 'none (caddr loc)))
+                                                      false)))))
```

Comment:
Okay, I've now added a `Bool` flag to `GeneratedFunctionStub` that inference can check to decide when to expand the generator. Just to get started, I'm hardcoding the flag to `false`, but now I need to add a mechanism for actually setting it. 

@JeffBezanson What would be the most reasonable way to add this setting? On the front end, I could add an extra argument to the `@generated` macro, or add a `@generated_early` macro, etc. I'm also wondering what the easiest/cleanest way is to propagate that setting to here. I guess I could add a meta node to the expression returned from `@generated` and then destructure it here?

...I'm a noob when it comes to Julia's parser 😛